docs/modules/ROOT/pages/deploy-management-center.adoc

```diff
@@ -0,0 +1,210 @@
+----
+include::ROOT:example$/management-center.yaml[]
+----
+
+. Apply it with the following command to start Management Center.
++
+.For Kubernetes
+[source,shell]
+----
+kubectl apply -f management-center.yaml
+----
++
+.For Openshift
+[source,shell]
+----
+oc apply -f management-center.yaml
+----
+
+
+. After a moment, you can verify that Management Center is up and running by checking the Management Center logs.
++
+.For Kubernetes
+[source,shell]
+----
+kubectl logs pod/managementcenter-sample-0
+----
++
+.For Openshift
+[source,shell]
+----
+oc logs pod/managementcenter-sample-0
+----
+
+```
+2021-08-26 15:21:04,842 [ INFO] [MC-Client-dev.lifecycle-1] [c.h.w.s.MCClientManager]: MC Client connected to cluster dev.
+2021-08-26 15:21:05,241 [ INFO] [MC-Client-dev.event-1] [c.h.w.s.MCClientManager]: Started communication with member: Member [10.36.8.3]:5701 - ccf31703-de3b-4094-9faf-7b5d0dc145b2
+2021-08-26 15:21:05,245 [ INFO] [MC-Client-dev.event-1] [c.h.w.s.MCClientManager]: Started communication with member: Member [10.36.7.2]:5701 - e75bd6e2-de4b-4360-8113-040773d858b7
+2021-08-26 15:21:05,251 [ INFO] [MC-Client-dev.event-1] [c.h.w.s.MCClientManager]: Started communication with member: Member [10.36.6.2]:5701 - c3d105d2-0bca-4a66-8519-1cacffc05c98
+2021-08-26 15:21:07,234 [ INFO] [main] [c.h.w.Launcher]: Hazelcast Management Center successfully started at http://localhost:8080/
+```
+
+--
+Enterprise::
++
+--
+To enable some features or dashboard at Management Center, you need a license key. If you don't have a license key, you can request one from the link:http://trialrequest.hazelcast.com/[Hazelcast website].
+
+. Create a Kubernetes secret to hold your license key.
++
+.For Kubernetes
+[source,shell]
+----
+kubectl create secret generic hazelcast-license-key --from-literal=license-key=<YOUR LICENSE KEY>
+----
++
+.For Openshift
+[source,shell]
+----
+oc create secret generic hazelcast-license-key --from-literal=license-key=<YOUR LICENSE KEY>
+----
+
+. Create the `ManagementCenter` custom resource file and name it `management-center.yaml`.
++
+[source,yaml,subs="attributes+"]
+----
+include::ROOT:example$/management-center-with-license-key.yaml[]
+----
+
+WARNING: StatefulSet does not support updates to `volumeClaimTemplates` field, so `persistence` field should be set only at the creation of the custom resource. Any update to the `persistence` field will not affect the Management Center.
+
+NOTE: By default, xref:management-center:deploy-manage:configuring.adoc#where-management-center-saves-data[Management Center data] is persisted to the `/data` directory. If you want to use an existing `PersistentVolumeClaim`, set its name in the `.spec.persistence.existingVolumeClaimName`field in the Management Center custom resource.
+
+WARNING: `hazelcastClusters` field does not support deleting clusters from the custom resource. If you want to remove a cluster from the Management Center, you can do it from the Management Center UI.
```

Comment:
You moved these warnings and note to the according pages. I think we can remove them from this page. No need for duplication IMHO.